ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -219,6 +219,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-ovn-upgrade,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I would make this false, and then we only need to run it when we think we are making changes that will affect the feature, WDYT?